Spotify UI/lib/helpers/size.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+getHeightOfDevice(context) => MediaQuery.of(context).size.height;
+getWidthOfDevice(context) => MediaQuery.of(context).size.width;
+renderSizedBox({width, height}) => SizedBox(
+      height: height,
+      width: width,
+    );
```

Comment:
This might be unconvenient as it drops the const constructor that SizedBox has. It helps a lot on the performance, because the const widgets wont be rebuilt when setState is called